<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorPickerWidget</class>
 <widget class="QWidget" name="ColorPickerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Kind:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="tristimulusRadioButton">
       <property name="text">
        <string>Tristimulus (RGB)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="spectralRadioButton">
       <property name="text">
        <string>Spectral</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="stackedWidgetPage1" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="TristimulusColorPicker" name="tristimulusPicker" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="stackedWidgetPage2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="SpectralColorPicker" name="spectralPicker" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TristimulusColorPicker</class>
   <extends>QWidget</extends>
   <header location="global">tristimuluscolorpicker.hh</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SpectralColorPicker</class>
   <extends>QWidget</extends>
   <header location="global">spectralcolorpicker.hh</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
